server/core/src/https/middleware/mod.rs

```diff
@@ -31,6 +34,41 @@ pub struct KOpId {
     pub uat: Option<String>,
 }
 
+/// Ensure the status code is 200..=299
+#[cfg(debug_assertions)]
+fn from_200_to_299(status: http::StatusCode) -> bool {
+    status.as_u16() >= 200 && status.as_u16() <= 299
+}
+
+#[test]
+fn test_from_200_to_299() {
+    assert!(from_200_to_299(http::StatusCode::OK));
+    assert!(from_200_to_299(http::StatusCode::IM_USED));
+    assert!(!from_200_to_299(http::StatusCode::BAD_REQUEST));
+    assert!(!from_200_to_299(http::StatusCode::INTERNAL_SERVER_ERROR));
+}
+
+#[cfg(debug_assertions)]
+/// This is a debug middleware to ensure that /v1/ endpoints only return JSON
```

Comment:
Oooo I really like this idea too. Good stuff !

Reply:
I was like "how do I write tests for this... oh, let's just make it part of the routing layer" :smile: